### PR TITLE
Feature/dapp 178

### PR DESCRIPTION
**Pull request by mendesmcmg**
Adds billing information section

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] It is urgent
- [ ] <span style="color:#d44037">Dangerous change</span>

### CODE REVIEW

File: src/lib/maskForTaxId/index.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Languages } from "types/enums/Languages";
+
+export const mask = (v: string, language: string) => {
```
Comment (NickNish09) [inline src/lib/maskForTaxId/index.ts:3]: maybe we could use some more mnemonic names to the variables `v` and `c`. What do you think?

File: src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { useTranslation } from "react-i18next";
+import { useEffect, useState } from "react";
+import InputAutoComplete from "components/atomics/inputs/InputAutoComplete";
+import { Languages } from "types/enums/Languages";
+import { useLanguage } from "hooks/useLanguage";
+import { countryList } from "./countryList";
+import { mask } from "../../../../../lib/maskForTaxId";
```
Comment (NickNish09) [inline src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.tsx:7]: I think here we can import it using the absolute path `lib/maskForTaxId`

File: src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  const [state, setState] = useState("");
+  const [city, setCity] = useState("");
+  const [taxId, setTaxId] = useState("");
+
+  const handleChangeMask = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+    setTaxId(mask(value, currentLang));
+  };
+
+  useEffect(() => {}, [taxId, state, city, country]);
```
Comment (NickNish09) [inline src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.tsx:28]: Is this being used?

File: src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+  return (
+    <S.BillingInformationSectionContainer>
+      <S.Title>{t("title")}</S.Title>
+
+      <S.Form>
+        <InputAutoComplete
```
Comment (mendesmcmg) [inline src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.tsx:35]: nome dos inputs

File: src/lib/maskForTaxId/index.ts (hunk 1 above)
Comment (mendesmcmg) [reply src/lib/maskForTaxId/index.ts:3]: good idea

File: src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.tsx (hunk 3 above)
Comment (mendesmcmg) [reply src/pages/promoters/SupportFundPage/CardSection/BillingInformationSection/index.tsx:28]: not yet, but I'm leaving it ready for when we do the back-end connection

**Review by NickNish09 — APPROVED** [review]

**Pull request merged**